src/rust/ensogl/lib/list-view/src/entry.rs

```diff
@@ -139,7 +139,9 @@ impl Entry {
         let display_object = display::object::Instance::new(logger);
         display_object.add_child(&label);
         label.set_position_xy(Vector2(PADDING + ICON_SIZE + ICON_LABEL_GAP, LABEL_SIZE/2.0));
-        label.set_default_color(color::Rgba::new(1.0,1.0,1.0,0.7));
+        let styles     = StyleWatch::new(&app.display.scene().style_sheet);
```

Comment:
You told me that styles do not change after ou press kb shortcut. Im not sure about other places, but here for example, you create a style watch utility, you query the color, and ... you remove the style watch utility ;) Moreover, the style watch utility has a function allowing us to register callback which tells - what should happen when style changes, which is not used here. Anyway, I dont think that `StyleWatch` is suitable here (it was designed as internal tool for shape system), so jsut add here fixme comment (and in other similar places) telling that the style watch i s not used propelry, is being dropped and this needs to be fixed.